builder-support/debian/recursor/debian-buster/rules

```diff
@@ -60,5 +65,5 @@ override_dh_gencontrol:
 
 override_dh_fixperms:
 	dh_fixperms
-        # these files often contain passwords. 640 as it is chowned to root:pdns
+# these files often contain passwords. 640 as it is chowned to root:pdns
```

Comment:
It looks like this bit came from the auth packaging, but recursor.conf can never have passwords, right?

Reply:
There's the webserver's credentials